core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/SharedServiceConfigProvider.java

```diff
@@ -86,6 +99,7 @@ private boolean decorateStackWithConfigs(Stack publicStack, Optional<DatalakeRes
         if (datalakeResource.isPresent()) {
```

Comment:
How will this be removed in the next phase?

Reply:
https://github.com/hortonworks/cloudbreak/blob/cb12835/core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/SharedServiceConfigProvider.java

like this:
`
    public Cluster configureCluster(@Nonnull Cluster requestedCluster, User user, Workspace workspace) {
      
      
                Objects.requireNonNull(requestedCluster);
      
      
                Stack stack = requestedCluster.getStack();
      
      
                if (!Strings.isNullOrEmpty(stack.getDatalakeCrn())) {
      
      
                    Stack datalakeStack = stackService.getByCrn(stack.getDatalakeCrn());
      
      
                    if (stack != null) {
      
      
                        Set&lt;RDSConfig&gt; rdsConfigs = rdsConfigService.findByClusterId(datalakeStack.getCluster().getId());
      
      
                        setupRds(requestedCluster, rdsConfigs);
      
      
                        setupStoragePath(requestedCluster, datalakeStack);
      
      
                    }
      
      
                }
      
      
                return requestedCluster;
      
      
            }`

Reply:
I am not sure understand completely the question. I referenced the  next phase branch